go.mod

```diff
@@ -117,6 +116,7 @@ require (
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/gomarkdown/markdown v0.0.0-20220905174103-7b278df48cfb // indirect
+	github.com/hashicorp/go-multierror v1.1.1 // indirect
```

Comment:
Toto je uz nepriama zavislost:
- Rozmyslam, ze by som `errors` rpesunul do `go-utils` a pouzili by sa aj v `go-client`, ale to mozno niekedy neskor.

Reply:
Teraz je to kompatibilny `hashicorp/go-multierror` ma rovnake (dolezite) metody ako nas kod, takze to funguje aj spolu.